hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DefaultSource.scala

```diff
@@ -177,7 +177,7 @@ class DefaultSource extends RelationProvider
       outputMode)
   }
 
-  override def shortName(): String = "hudi"
+  override def shortName(): String = "hudi_v1"
```

Comment:
@leesf i suppose this refactoring PR not meant to include this change?

Reply:
If not change the format, it would conflict with hudi-spark2/hudi-spark3.1.x/hudi-spark3 module format.

Reply:
would it conflict? Given we are extending `DefaultSource` and overriding `shortName()`?

Reply:
it is because in hudi-spark-bundle module. I used `<transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
                  <resource>META-INF/services/org.apache.spark.sql.sources.DataSourceRegister</resource>
                </transformer>` to append the formats(hudi_v1 and hudi) in `DataSourceRegister` file, so it will conflict if not change the format. As to the PR itself, we do not need to change the format to `hudi_v1` and not use AppendingTransformer. But when implementing V2 codepath, I find it difficult to handle the incremental bootstrap table(https://github.com/apache/hudi/blob/ac8d014a0602e3c499771f3313f0f88de57cdda1/hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/IncrementalRelation.scala#L174) as it will pass schema to Spark and difficult to handle in v2 codepath, after finding a good way to handle it, we would definitely delete the hudi_v1 format here.